.github/workflows/tests.yml

```diff
@@ -25,11 +25,15 @@ jobs:
       
       - run: go version
 
+      - name: install lambda runtime interface emulator
+        run: curl -L -o /usr/local/bin/aws-lambda-rie https://github.com/aws/aws-lambda-runtime-interface-emulator/releases/download/v1.6/aws-lambda-rie-x86_64
```

Comment:
change `v1.6` to `latest` in order to always test against the most recent release - eg. URL from RIE readme:
```
https://github.com/aws/aws-lambda-runtime-interface-emulator/releases/latest/download/aws-lambda-rie-x86_64
```

Reply:
will do 👍

Reply:
```
$ curl -L https://github.com/aws/aws-lambda-runtime-interface-emulator/releases/download/latest/aws-lambda-rie-x86_64
Not Found
```

Reply:
Oh oops no arch suffix for x86_64 apparently

https://github.com/aws/aws-lambda-runtime-interface-emulator/releases/latest/download/aws-lambda-rie

Reply:
Actually it seems to be a different url ordering for latest

blahblah/latest/download/

Not 

blahblah/download/v1.6/

🤷‍♂️